spec/features/reviews/02_create_review_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature 'add a new review', %{
+  As a user
+  I want to add a review
+  So that I can post my review
+  } do
+  # ACCEPTANCE CRITERIA
+  # * As an authenticated user, I can add a new review
+
+  scenario 'authenticated user adds reviews on park show page' do
+    park = create(:park)
+
+    user = create(:user)
+
+    visit root_path
```

Comment:
Throw this line at the beginning of you `#sign_in_as` method so you don't have to do this for every test.